src/service/tag-service.js

```diff
@@ -0,0 +1,22 @@
+import TagRepogitory from '../repository/tag-repository.js';
+
+class TagService {
+  async getTags(page, limit, order, search) {
+    const where = {
+      name: { contains: search, mode: 'insensitive' },
+    };
+
```

Comment:
```suggestion
    const totalCount = await TagRepository.tagsCount(where);

    if (totalCount === 0) {
      return {tags; [], totalCount: 0};
    }
```
리포지토리의 태그 카운트에서 결과값을 반환 받고 카운트가 0을 반환하면 빈 배열과 카운트: 0 을 반환합니다,